index.html

```diff
@@ -0,0 +1,244 @@
+}
+
+h4 {
+  font-family: 'Raleway';
+  font-weight: bold 800;
+  color: #323232;
+}
+
+.formbox {
+  position: relative;
+  width: 80%;
+  height: 80%;
+  margin: 50px;
+  margin-left: 100px;
+  margin-top: 30px;
+  background-color: white;
+  border: 20px solid;
+}
+
+.formint {
+  margin: 30px;
+  margin-top: 20px;
+}
+
+.inputd {
+  padding: 12px;
+  width: 80%;
+  margin-bottom: 20px;
+  border-radius: 2px;
+  margin-left: 33px;
+  border: 2px solid #EFEFEF;
+  color: grey;
+  font-weight: bold;
+  letter-spacing: -0.5px;
+  font-family: 'Raleway';
+  font-size: 14px;
+}
+
+.loginform {
+  top: 30px;
+  position: relative;
+  width: 30%;
+  height: 420px;
+  margin-left: 80px;
+  color: #fff;
+  background: #fff;
+  justify-content: center;
+}
+
+.tagline {
+  margin-left: 33px;
+  font-family: 'Raleway';
+  margin-top: 40px
+}
+
+.loginform::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: linear-gradient( white, grey, grey);
+  z-index: -2;
+  filter: blur(20px);
+}
+
+label {
+  color: black;
+  font-size: 13px;
+  padding-bottom: 10px;
+  font-weight: bold;
+  font-family: 'Raleway';
+}
+
+.btn {
+  padding: 10px;
+  width: 25%;
+  margin-right: 5px;
+  margin-top: 10px; 
+  font-family: 'Raleway';
+  border: 2px solid #EFEFEF;
+  background-color: white;
+  font-weight: bold;
+  color: grey !important;
+}
+
+.startbtn {
+  background-color: rgb(30, 172, 178);
+  padding: 15px;
+  color: white!important;
+  width: 80%;
+  margin-left: 33px;
+  border: none;
+  border-radius: 6px;
+  font-size: 16px;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+}
+
+.callus {
+  margin-left: 93px;
+  font-weight: bold;
+  font-size: 12px;
+  color: grey;
+  margin-top: 25px;
+  font-family: 'Roboto';
+}
+</style>
+</head>
+<title>Excercise2</title>
+<body>
+  <div class="outer">
+  <div  class="dev">
+    <h3 class="heading">Why Loconav's Vehicle Tracking System ?</h3>
+    <div>
+      <div class="row">
+        <div class="column" style="background-color: #FFF">
+          <img src="image1.png" class="vehicle-track-image">
+          <h4 class="headinternal">INTELLIGENT VEHICLE TRACKING</h4>
+          <p class="description"> Wheather you are single vehicle or a <span style="color: #6399C7">fleet owner</span>, Loconav is determined to protect and monitor your asset from theft or harm</span></p>
+        </div>
+        <div class="column" style="background-color: #FFF" >
+          <img src="image4.png" class="vehicle-lock-image" >
+          <h4 class="headinternal">
+          VEHICLE LOCK</h3>
+          <p class="description"> Turn on vehicle lock from Loconav's <span style="color: #6399C7">mobile app</span> and be assured that your vehicle won't start without your permission</p>
+        </div>
+        <div class="column" style="background-color: #FFF">
+          <img src="image2.png" class="live-alert-image" >
+          <h4 class="headinternal">
+          LIVE ALERTS</h3>

```

Comment:
please fix the incorrect closing tag here

Reply:
corrected